armi/reactor/blueprints/__init__.py

```diff
@@ -185,6 +186,9 @@ class Blueprints(yamlize.Object, metaclass=_BlueprintsPluginCollector):
     )
     systemDesigns = yamlize.Attribute(key="systems", type=Systems, default=None)
     gridDesigns = yamlize.Attribute(key="grids", type=Grids, default=None)
+    particleFuelDesigns = yamlize.Attribute(
+        key="particle fuel", type=ParticleFuelKeyedList, default=None
+    )
```

Comment:
I'm hoping we can come up with a slightly more generic name for this that would still meet the need that you have. 

We have been having some related internal discussions that I have now put in #503 and #504 that I think are very related to this. 

In this case, one could imagine having a section at this level called something like `Mixtures`.